### DISCUSSION TITLE
Firewall rules to allow signal-cli to send messages

### DISCUSSION THREAD

**Discussion by patrickchan3721** [AsamK/signal-cli#802] [General]
My server is behind firewall.

1. Which tcp and udp port should be allowed?

2. Can signal-cli use proxy to connect to internet?

Thanks!

**Comment by AsamK** [comment]
signal-cli doesn't require any incoming ports, and only https (tcp 443) for outgoing traffic.
Using a proxy might work by setting some java properties: https://github.com/AsamK/signal-cli/issues/79#issuecomment-922374818
